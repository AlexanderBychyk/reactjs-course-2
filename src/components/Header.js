import React, { Component } from 'react';

import SiteLogo from './SiteLogo';
import AddMovie from './AddMovie';
import SearchBar from './SearchBar'

export default class Header extends Component {
  render() {
    return (
      <div className="headerBlock">
        <div className="container">
          <SiteLogo />
          <AddMovie />
        </div>
        <p className="headerLabel">FIND YOUR MOVIE</p>
        <SearchBar />
      </div>
    );
  }
};
