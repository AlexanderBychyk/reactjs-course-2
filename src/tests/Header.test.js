import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../components/Header';

it('renders <Header /> without crashing', () => {
  const component = renderer.create(<Header />).toJSON();
  expect(component).toMatchSnapshot();
});
