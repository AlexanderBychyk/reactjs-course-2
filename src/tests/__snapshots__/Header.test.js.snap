// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Header /> without crashing 1`] = `
<div>
  <h1>
    Hello World!
  </h1>
</div>
`;
